src/components/HeadFooter/HeadFooter.js

```diff
@@ -37,19 +37,29 @@ const overwrittenTheme = createTheme({
 	}
 });
 
-const useStyles = makeStyles(() => ({
-	wrapper: {
-		minHeight: '100%',
-		display: 'flex',
-		flexDirection: 'column'
-	},
-	headFooter: {
-		flexShrink: 0
-	},
-	main: {
-		flexGrow: 1
-	}
-}));
+const mobileAdaptiveThreshold = theme => theme.breakpoints.values.sm * 1.05;
+
+const useStyles = makeStyles(theme => {
+	const menuBarAdaptiveThreshold = mobileAdaptiveThreshold(theme);
+	return {
+		wrapper: {
+			minHeight: '100%',
+			display: 'flex',
+			flexDirection: 'column'
+		},
+		headFooter: {
+			flexShrink: 0
+		},
+		main: {
+			flexGrow: 1,
+			paddingTop: theme.spacing(8),
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				paddingTop: theme.spacing(7)
+			}
```

Comment:
Also out of curiosity, where did you get spacing 7 for mobile and 8 for desktop? (Or at least that's how I'm understanding this piece of code)

Reply:
inspect element and looked at the pixels LOL (theme.spacing(1) is 8 pixels) i think there's a better way to do this entire sticky navbar but in essense this is how it would've worked

Reply:
Well I have no clue how to make it better so nice!